llvm/lib/MC/MCSectionGOFF.cpp

```diff
@@ -0,0 +1,141 @@
+//===- MCSectionGOFF.cpp - GOFF Code Section Representation ---------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "llvm/MC/MCSectionGOFF.h"
+#include "llvm/BinaryFormat/GOFF.h"
+#include "llvm/Support/raw_ostream.h"
+
+using namespace llvm;
+
+namespace {
+void emitCATTR(raw_ostream &OS, StringRef Name, GOFF::ESDRmode Rmode,
```

Comment:
use `static` https://llvm.org/docs/CodingStandards.html#restrict-visibility

Reply:
Remove the anonymous namespace, and added `static`.